src/theory/strings/core_solver.h

```diff
@@ -522,6 +524,8 @@ class CoreSolver : protected EnvObj
    * on the ordering described in checkCycles.
    */
   std::vector<Node> d_strings_eqc;
+  /** The relevant disequalities */
```

Comment:
```suggestion
  /** The relevant disequalities for `to_code`*/
```
?

Reply:
Thanks for the changes. Only this one is left. If it is correct, I think it would make things clearer.

Reply:
This is documented above the public method to access this (see `getRelevantDeq`).